### DISCUSSION TITLE
请教如何使用SASS

### DISCUSSION THREAD

**Discussion by zengdaming** [InfpHub/Raptor#34] [Q&A]
大佬您好，
我看到你的项目中用到了SASS，请问能否指点一下是怎么做的，或者提供相关的资料。作为QT新手的我，在没有找到相关的处理的方法。网上也没搜到对应的资料。
我也想在自己的项目中，用上SASS

**Comment by InfpHub** [comment]
> 大佬您好，
> 我看到你的项目中用到了SASS，请问能否指点一下是怎么做的，或者提供相关的资料。作为QT新手的我，在没有找到相关的处理的方法。网上也没搜到对应的资料。
> 我也想在自己的项目中，用上SASS

该库为 C 语言库，您可以搜索 libsass 编译即可使用。

**Reply by zengdaming** [reply]
非常感谢指教。因为一直比较忙，迟来了感谢
